DESCRIPTION

```diff
@@ -35,4 +32,10 @@ Imports:
     tidyselect,
     tibble,
     viridis
+Suggests: 
+    devtools,
```

Comment:
This one is only recommended if someone would install ifadv via devtools, right? But then, if we put ifadv in Suggests as well, we no longer need devtools...

Reply:
Sounds good!
```suggestion
```